### PR TITLE
Audit log doc update for clarity

### PR DESCRIPTION
**Pull request by abhinavg6**
## Description

Making a few updates to the audit logs doc to provide clarity regarding supported deployment types, modes of getting the logs, log retention etc. Also removed the private preview label.

## Ticket

NA

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings. - still getting the unrelated TypeError: Invalid URL errors.
- [X] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/hosting/audit-logging.md [hunk 1]
```diff
@@ -4,10 +4,16 @@ displayed_sidebar: default
 
 
 # Audit logs
-Use audit logs to track and understand activity within your team. Audit logs sync to your bucket store every 10 minutes. Optionally, download your audit logs and view them with your preferred tool, such as [Pandas](https://pandas.pydata.org/docs/index.html), [BigQuery](https://cloud.google.com/bigquery), and more.
+Use W&B Server audit logs to track user activity within your teams, and to conform to your enterprise governance requirements. If you've a **Self-managed** W&B Server deployment, or use the [secure storage connector (BYOB)](./secure-storage-connector.md) with your **Dedicated Cloud** deployment, JSON-formatted audit logs sync to your instance-level bucket every 10 minutes. Else when you have a **Dedicated Cloud** deployment without the secure storage connector (BYOB), [instance admins](./manage-users.md#instance-admins) can access audit logs using an API (see below).
```
Comment (yogeshg) [inline docs/guides/hosting/audit-logging.md]:
```suggestion
Use W&B Server audit logs to track user activity within your teams, and to conform to your enterprise governance requirements. If you have a **Self-managed** W&B Server deployment, and use the [secure storage connector (BYOB)](./secure-storage-connector.md) with your **Dedicated Cloud** deployment, JSON-formatted audit logs sync to your instance-level bucket every 10 minutes. Else when you have a **Dedicated Cloud** deployment without the secure storage connector (BYOB), [instance admins](./manage-users.md#instance-admins) can access audit logs using an API (see below).
```

File: docs/guides/hosting/audit-logging.md [hunk 2]
```diff
@@ -35,24 +41,22 @@ The following table lists all the different keys that might be present in your a
...
+To fetch the audit logs for your W&B server instance using API, an instance admin can follow these steps:
+1. Construct the full API endpoint using a combination of the base endpoint `<wandb-server-url>/admin/audit_logs` and the following URL parameters:
+    - `numDays` : logs will be fetched starting from `today - numdays` to most recent; defaults to `0` i.e. logs will be returned only for `today`
     - `anonymize` : if set to `true`, remove any PII; defaults to `false`
+2. Execute HTTP GET request on the constructed full API endpoint, either by directly running it within a modern browser, or by using a tool like [Postman](https://www.postman.com/downloads/), [HTTPie](https://httpie.io/), cURL command or more.
 
-Note that only W&B server admins are allowed to request this information. If you are not an admin you will an authentication error. The response contains new-line separated JSON objects. Objects will have fields described in the schema.
+If your W&B Server instance URL is `https://mycompany.wandb.io` and you would like to get anonymized audit logs for activity within the last week, your API endpoint will be `https://mycompany.wandb.io?numDays=7&anonymize=true`.
```
Comment (yogeshg) [inline docs/guides/hosting/audit-logging.md]:
```suggestion
If your W&B Server instance URL is `https://mycompany.wandb.io` and you would like to get (pseudo) anonymized audit logs for activity within the last week, your API endpoint will be `https://mycompany.wandb.io?numDays=7&anonymize=true`.
```

Comment (yogeshg) [reply docs/guides/hosting/audit-logging.md]: we can even say "would like to get audit logs without PII".
more context here: https://weightsandbiases.slack.com/archives/C02K03U0Z18/p1687966119296129

File: docs/guides/hosting/audit-logging.md (hunk 1 above)
Comment (abhinavg6) [reply docs/guides/hosting/audit-logging.md]: @yogeshg - That should be an "or" not an "and". Both relate to two different types of deployments. "And" would indicate that both should be true which is not the case. I'll make the nit change for you've to you have.

**Review by yogeshg — APPROVED** [review]
Tysm! 🙏

File: docs/guides/hosting/audit-logging.md (hunk 2 above)
Comment (abhinavg6) [reply docs/guides/hosting/audit-logging.md]: Ok, I'll change this. Thanks.

File: docs/guides/hosting/audit-logging.md (hunk 1 above)
Comment (yogeshg) [reply docs/guides/hosting/audit-logging.md]: Ah, I see, I re-read, makes sense.

**Comment by abhinavg6** [comment]
Thanks @yogeshg - I've added another commit for the nit changes requested by you.

File: docs/guides/hosting/audit-logging.md [hunk 3]
```diff
@@ -4,10 +4,16 @@ displayed_sidebar: default
 
 
 # Audit logs
-Use audit logs to track and understand activity within your team. Audit logs sync to your bucket store every 10 minutes. Optionally, download your audit logs and view them with your preferred tool, such as [Pandas](https://pandas.pydata.org/docs/index.html), [BigQuery](https://cloud.google.com/bigquery), and more.
+Use W&B Server audit logs to track user activity within your teams, and to conform to your enterprise governance requirements. If you have a **Self-managed** W&B Server deployment, or use the [secure storage connector (BYOB)](./secure-storage-connector.md) with your **Dedicated Cloud** deployment, JSON-formatted audit logs sync to your instance-level bucket every 10 minutes. Else when you have a **Dedicated Cloud** deployment without the secure storage connector (BYOB), [instance admins](./manage-users.md#instance-admins) can access audit logs using an API (see below).
```
Comment (ngrayluna) [inline docs/guides/hosting/audit-logging.md]: I found these two paragraphs a little confusing. Locally I changed it to this:

---
Use W&B Server audit logs to track user activity within your teams, and to conform to your enterprise governance requirements. Based on your hosting type, there are two ways to access audit logs:


| Deployment| How to access|
| ------ | ----- |
|**Self-managed** W&B Server deployment| JSON-formatted audit logs sync to your instance-level bucket every 10|
|**Dedicated Cloud** deployment with [secure storage connector (BYOB)](./secure-storage-connector.md) | JSON-formatted audit logs sync to your instance-level bucket every 10 minutes |
| **Dedicated Cloud** deployment without secure storage connector (BYOB)| [Instance admins](./manage-users.md#instance-admins) [can access audit logs with APIs](#fetch-audit-logs-using-api). | 

Analyze audit logs with your preferred tools, such as [Pandas](https://pandas.pydata.org/docs/index.html), [Amazon Redshift](https://aws.amazon.com/redshift/), [Google BigQuery](https://cloud.google.com/bigquery), [Microsoft Fabric](https://www.microsoft.com/en-us/microsoft-fabric), and more regardless of how you access your audit logs. You might need to transform JSON-formatted audit logs into a format relevant to the tool for the third-party tools. Information on how to transform your audit logs is outside the scope of W&B documentation.
Screen Shot 2023-12-26 at 4 38 48 PM

File: docs/guides/hosting/audit-logging.md [hunk 4]
```diff
@@ -35,24 +41,22 @@ The following table lists all the different keys that might be present in your a
...
-Personally identifiable information (PII) like email ids, project, team and report names are returned only by the endpoint, and can be turned off as [described below](#view-audit-logs).
+Personally identifiable information (PII) like email ids, project, team and report names are available only using the API endpoint option, and can be turned off as [described below](#fetch-audit-logs-using-api).
 
-## View audit logs
-To view the audit logs for your W&B server instance, follow these steps:
-1. Admin users can go to `<wandb-server-url>/admin/audit_logs`
-2. Pass in the following URL parameters:
-    - `numDays` : logs will be fetch starting from `today - numdays` to most recent; defaults to `0`
+## Fetch audit logs using API
+To fetch the audit logs for your W&B server instance using API, an instance admin can follow these steps:
```
Comment (ngrayluna) [inline docs/guides/hosting/audit-logging.md]: "To fetch the audit logs for your W&B server instance using API, an instance admin can follow these steps:" -> "Fetch audit logs for your W&B server instance using the following APIs:"

**Review by ngrayluna — COMMENTED** [review]
Left some comments/suggestions for verbage.

File: docs/guides/hosting/audit-logging.md (hunk 4 above)
Comment (abhinavg6) [reply docs/guides/hosting/audit-logging.md]: Sure, I can change that accordingly.

File: docs/guides/hosting/audit-logging.md (hunk 3 above)
Comment (abhinavg6) [reply docs/guides/hosting/audit-logging.md]: Yeah, makes sense to convert that into a table. Let me do that.

**Comment by abhinavg6** [comment]
Thanks @ngrayluna . I've made changes as per the feedback.

**Review by ngrayluna — APPROVED** [review]
Thanks Abhinav!

**Pull request merged**